source/_components/egardia.markdown

```diff
@@ -0,0 +1,67 @@
+---
+layout: page
+title: "Egardia"
+description: "Instructions how to setup Egardia / Woonveilig within Home Assistant."
+date: 2018-03-02 09:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Hub
+ha_release: 0.65
+logo: egardia.png
+---
+
+The `egardia` platform enables the ability to control an [Egardia](http://egardia.com/)/[Woonveilig](http://woonveilig.nl) control panel. These alarm panels are known under different brand names across the world, including Woonveilig in the Netherlands. This was tested on the WL-1716, GATE-01, GATE-02 and GATE-03 versions of the Egardia/Woonveilig platform. Not only will you integrate your alarm control panel, supported sensors (door contacts at this moment) will be added automatically (hidden by default).
+
+You will need to know the IP of your alarm panel on your local network. Test if you can login to the panel by browsing to the IP address and log in using your Egardia/Woonveilig account.
+
+To enable the integration with your alarm panel, add the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+egardia:
+  host: YOUR_HOST
+  username: YOUR_USERNAME
+  password: YOUR_PASSWORD
+```
+
+Configuration variables:
```

Comment:
Please update this to use the configuration tag:
https://home-assistant.io/developers/documentation/create_page/#configuration

Reply:
I want to do the release right now so going ahead and merge this. @jeroenterheerdt please address this comment in a new PR.

Reply:
Sure, will do.